src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,36 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+/**
+ * Represents a Task with an additional dateTime field storing a deadline
+ */
+public class Deadline extends Task {
+    private LocalDate dateTime;
```

Comment:
does your datetime actually store the time?

Reply:
No. I will rename ```dateTime``` to ```date``` for accuracy.